src/CommitmentsV2/SFA.DAS.CommitmentsV2.Messages/Events/IApprenticeshipCreatedEvent.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using SFA.DAS.CommitmentsV2.Types;
+
+namespace SFA.DAS.CommitmentsV2.Messages.Events
+{
+    public interface IApprenticeshipCreatedEvent
+    {
+        long ApprenticeshipId { get; set; }
+        DateTime CreatedOn { get; set; }
+        string Uln { get; set; }
+        long ProviderId { get; set; }
+        long AccountId { get; set; }
+        string AccountLegalEntityPublicHashedId { get; set; }
+        string LegalEntityName { get; set; }
+        DateTime StartDate { get; set; }
+        DateTime EndDate { get; set; }
+        PriceEpisode[] PriceEpisodes { get; set; }
+        TrainingType TrainingType { get; set; }
+        string TrainingCode { get; set; }
+        long? TransferSenderId { get; set; }
+    }
+
+    public class PriceEpisode
```

Comment:
Could this not be an interface as well?

Reply:
I actually need to new this up. Therefore will have to have the implementation and the interface. So doesn't seem to be any value in making this element an interface